io.github.jliljebl.Flowblade.yaml

```diff
@@ -99,9 +99,28 @@ modules:
       - /bin
     sources:
       - type: archive
-        url: https://github.com/python-pillow/Pillow/archive/5.3.0.tar.gz
-        sha256: 9de4853d8b640c0185665735a1c3266b6d51636b3a3d8e3ea7876c485194b6e1
+        url: https://github.com/python-pillow/Pillow/archive/refs/tags/10.2.0.tar.gz
+        sha256: fe695f6fa8bbc341b9044b6553a32d84cf6d6ea0de104396aece85e454c7cbc2 
 
+  - name: usb1
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=/app --root=/
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/vpelletier/python-libusb1/releases/download/3.1.0/libusb1-3.1.0.tar.gz
+        sha256: 4ee9b0a55f8bd0b3ea7017ae919a6c1f439af742c4a4b04543c5fd7af89b828c
+
+  - name: usbrules
+    buildsystem: simple
+    build-commands:
+      - install -D 90-flowblade.rules /app/etc/udev/rules.d/90-flowblade.rules
```

Comment:
what will this do? there is no udev.

Reply:
http://jliljebl.github.io/flowblade/webhelp/advanced.html#6._Jog_/_Shuttle_Support

This is documentation by the feature author. They say that this file needs to added for app to see the USB devices. I don't have one so this is not tested, but I added the file with Flatpak so that at least that is not the problem.

Reply:
and there is no udev inside the sandbox. it does nothing.

Reply:
Anyway, let's get this Flatpak out for wider consumption and see what kind feedback I get.